package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "browser-support-checker",
+  "version": "1.0.0",
+  "description": "Lightweight browser support checker",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepare": "npm run build",
+    "build": "webpack --config ./config/webpack.prod.config.js",
+    "build:watch": "webpack --config ./config/webpack.prod.config.js --watch --info-verbosity verbose"
+  },
+  "keywords": [
+    "browser",
+    "support",
+    "check",
+    "lightweight"
+  ],
+  "author": "Ivan Katkov <ivan.katkov@taotesting.com>",
+  "license": "ISC",
+  "dependencies": {
+    "@oat-sa/browserslist-config-tao": "^0.3.0",
+    "browserslist-useragent": "^3.0.3",
+    "browserslist-useragent-regexp": "^3.0.0"
```

Comment:
this seems usefull, but to compile a regexp from the query rather than including it at runtime. It can be a way to reduce the bundle size